pkg/cluster/internal/delete/delete.go

```diff
@@ -33,17 +33,25 @@ func Cluster(logger log.Logger, p providers.Provider, name, explicitKubeconfigPa
 		return errors.Wrap(err, "error listing nodes")
 	}
 
-	kerr := kubeconfig.Remove(name, explicitKubeconfigPath)
+	l, kerr := kubeconfig.Remove(name, explicitKubeconfigPath)
+	if l {
+		logger.V(0).Infof("Removed context %q from kubeconfig", name)
+	}
```

Comment:
do we really need this?

Reply:
This is what was asked for by @BenTheElder

Reply:
I’m fine to take it out if that’s what we want to do.

Reply:
Alright, I removed it.